openproblems/tasks/_batch_integration/batch_integration_embed/README.md

```diff
@@ -19,13 +19,18 @@ This sub-task was taken from a
 methods](https://www.biorxiv.org/content/10.1101/2020.05.22.111161v2).
 
 ## API
+WARNING: other than most tasks, `adata.X` should contain log-normalized data.
+   This is the case as we are comparing the results of integration on the normalized
+   features pre- and post-integration and the data comes from different technologies.
```

Comment:
We're not actually comparing the normalized features here, are we? This is the embedding task, it shouldn't matter whether or not the method operates on the provided features or the raw counts.

Reply:
True, we're not, however we're calculating the embedding pre-integration on the normalized data

Reply:
I think we might be comparing this indirectly as we generate the embedding via PCA from the features in some cases

Reply:
the same goes for the graph task above.

Reply:
I see. Fair enough. The I think this warning should apply to all three tasks, but we need to clarify why (by explaining the above comments in the relevant READMEs)

Reply:
Yeah, might be good to just add that embeddings/graphs are calculated from these features.